python/docs/source/user_guide/bugbusting.ipynb

```diff
@@ -458,7 +458,7 @@
    "id": "7f01c836",
    "metadata": {},
    "source": [
-    "#### Python and Pandas UDF"
+    "#### Python/Pandas UDF"
```

Comment:
To be consistent with https://github.com/apache/spark/pull/50850/files#diff-0110ef9ebba0fe09d27b2fc7e9c7e393a3a7e7df73f008e3733843d421339614L644.